set.mm

```diff
@@ -837895,6 +838014,1624 @@ have GLB (expanded version).  (Contributed by Zhi Wang,
+      upeu4 $p |- ( ph -> Y ( <. F , G >. ( D UP E ) W ) N ) $=
+        ( co cfv vg vk vy vx vf cop cup wbr cv cco wceq chom wreu wral cbs wcel
+        ciso eqid uprcl2 uprcl4 cmpo cxp ccat funcrcl2 isofn fneq1d mpbird fnov
+        wfn sylib oveqd eleqtrd elmpocl2 uprcl3 uprcl5 isup2 eqtrd upeu2 simprd
+        syl simpld isup ) ALIDEUFJBCUGSSUHUAUIUBUILUCUIZESTIJLDTZUFWCDTZCUJTZSS
+        UKUBLWCBULTZSUMUAJWECULTZSUNUCBUOTZUNZAIJWDWHSUPZWJAUDUCWICUOTZBUEUAUBC
+        DEWGBUQTZWHGHIWFKLJUBWIURZWLURZWGURZWHURZWFURZABCDEHJKPUSZAWIBCDEHJKPWN
+        UTAGKLUDUCWIWIUDUIWCFSZVAZSZUPLWIUPAGKLFSZXBQAFXAKLAFWIWIVBZVIZFXAUKAXE
+        WMXDVIZABVCUPXFABCDEWSVDBVEVTAXDFWMNVFVGUDUCWIWIFVHVJVKVLUDUCWIWIWTKLXA
+        GXAURVMVTZAWLBCDEHJKPWOVNZABCDEWHHJKPWQVOAUDWIBUEUBCDEWGWHHWFJKWNWPWQWR
+        PVPWMURAGXCKLWMSQAFWMKLNVKVLAIGKLESTZHMSXIHJKDTUFWDWFSZSRAMXJXIHOVKVQVR
+        ZVSAUCWIWLBUAUBCDEWGWHIWFJLWNWOWPWQWRXHWSXGAWKWJXKWAWBVG $.
+    $}
+  $}
+
+  ${
+    $d B g k y $.  $d C g k y $.  $d F g k y $.  $d G g k y $.  $d H k $.
+    $d M g k y $.  $d O g k y $.  $d P g k y $.  $d W g k y $.  $d X g k y $.
+    $d g k ph y $.
+    oppcup.b $e |- B = ( Base ` D ) $.
+    oppcup.c $e |- C = ( Base ` E ) $.
+    oppcup.h $e |- H = ( Hom ` D ) $.
+    oppcup.j $e |- J = ( Hom ` E ) $.
+    oppcup.xb $e |- .xb = ( comp ` E ) $.
+    oppcup.w $e |- ( ph -> W e. C ) $.
+    oppcup.f $e |- ( ph -> F ( D Func E ) G ) $.
+    oppcup.x $e |- ( ph -> X e. B ) $.
+    oppcup.m $e |- ( ph -> M e. ( ( F ` X ) J W ) ) $.
+    oppcup.o $e |- O = ( oppCat ` D ) $.
+    oppcup.p $e |- P = ( oppCat ` E ) $.
+    $( The universal ` <. X , M >. ` from ` F ` to ` W ` is universal from
+       ` W ` to ` F ` in opposite categories.  (Contributed by Zhi Wang,
+       24-Sep-2025.) $)
+    oppcup $p |- ( ph -> ( X ( <. F , tpos G >. ( O UP P ) W ) M <->
+       A. y e. B A. g e. ( ( F ` y ) J W ) E! k e. ( y H X )
+       g = ( M ( <. ( F ` y ) , ( F ` X ) >. .xb W ) ( ( y G X ) ` k ) ) ) ) $=
+      ( ctpos cop cup co wbr cv cfv cco wceq chom wreu oppcbas funcoppc oppchom
+      wral eqid eleqtrrdi isup wcel wa ovtpos fveq1i oveq1i adantr cfunc funcf1
+      ffvelcdmd simpr oppcco eqtrid eqeq2d reueqbidv raleqbidv ralbidva bitrd
+      a1i ) AROKLUJZUKQPFULUMUMUNHUOZIUOZRBUOZWFUMZUPZOQRKUPZUKWIKUPZFUQUPZUMZU
+      MZURZIRWIPUSUPZUMZUTZHQWMFUSUPZUMZVDZBCVDWGOWHWIRLUMZUPZWMWLUKQGUMUMZURZI
+      WIRMUMZUTZHWMQNUMZVDZBCVDABCDPHIFKWFWRXAOWNQRCEPUHSVADJFUITVAWRVEXAVEWNVE
+      UDAEJFKLPUHUIUEVBUFAOWLQNUMQWLXAUMUGJNFQWLUBUIVCVFVGAXCXKBCAWICVHZVIZWTXI
+      HXBXJXBXJURXMJNFQWMUBUIVCWEXMWQXGIWSXHWSXHURXMEMPRWIUAUHVCWEXMWPXFWGXMWPX
+      EOWOUMXFWKXEOWOWHWJXDRWILVJVKVLXMDJGOXEFQWLWMTUCUIAQDVHXLUDVMXMCDRKXMCDEJ
+      KLSTAKLEJVNUMUNXLUEVMVOZARCVHXLUFVMVPXMCDWIKXNAXLVQVPVRVSVTWAWBWCWD $.
+  $}
+
+
+$(
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+  Natural transformations and the functor category
```

Comment:
What is a "natural transformation"?

Reply:
[Wikipedia](https://en.wikipedia.org/wiki/Natural_transformation) and [nLabs](https://ncatlab.org/nlab/show/natural+transformation) both have pages about natural transformations.